.github/actions/composer-install/action.yml

```diff
@@ -0,0 +1,56 @@
+name: Composer Install
+
+inputs:
+  OP_SERVICE_ACCOUNT_TOKEN:
+    required: true
+    description: 1Password service account token to use for populating composer auth.json
+  OP_VAULT:
```

Comment:
Thoughts on the name of these secrets? Should the be more verbose?

Reply:
> Secret names can only contain alphanumeric characters ([a-z], [A-Z], [0-9]) or underscores (_). Spaces are not allowed. Must start with a letter ([a-z], [A-Z]) or underscores (_).

😞

Reply:
`ONEPASS` would be more clear than `OP`.